judge/migrations/0104_webauthn.py

```diff
@@ -0,0 +1,30 @@
+# Generated by Django 2.2.12 on 2020-05-03 23:58
+
+import django.db.models.deletion
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('judge', '0103_contest_participation_tiebreak_field'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='profile',
+            name='is_webauthn_enabled',
+            field=models.BooleanField(default=False, help_text='check to enable WebAuthn-based two factor authentication', verbose_name='WebAuthn 2FA enabled'),
```

Comment:
```suggestion
            field=models.BooleanField(default=False, help_text='check to enable WebAuthn-based two-factor authentication', verbose_name='WebAuthn 2FA enabled'),
```

Reply:
Cannot accept suggestion because it's not also made to the model itself. Furthermore, TOTP needs to be fixed too.